src/components/Icons.tsx

```
@@ -0,0 +1,206 @@
+    />
+    {/* Revolut Text (top-left) */}
+    <Text
+      style={{
+        position: "absolute",
+        top: 8,
+        right: 50,
+        color: "#fff",
+        fontSize: 5,
+        fontWeight: "bold",
+      }}
+    >
```

ESLint

[error] Use <AppText> instead of <Text>. The <AppText> component is located in 'components/AppText'. We are using this because it has a custom font and some other features.

(no-restricted-syntax)